### PR TITLE
add foreman_userdata package [rpm]

### PR DESCRIPTION
**Pull request by timogoebel**
Please build for:

* [x] Nightly
* [x] 1.16
* [x] 1.15
* [x] 1.14

### CODE REVIEW

**Comment by GregSutcliffe** [comment]
This looks sane - @ehelms ?

File: rubygem-foreman_userdata/rubygem-foreman_userdata.spec [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+%global gem_name foreman_userdata
+%global plugin_name userdata
+
+Name: %{?scl_prefix}rubygem-%{gem_name}
+Version: 0.0.1
+Release: 1%{?foremandist}%{?dist}
+Summary: This plug-in adds support for serving user-data for cloud-init to The Foreman
+Group: Applications/Systems
+License: GPL-3
+URL: http://github.com/theforeman/foreman_userdata
```
Comment (ekohl) [inline rubygem-foreman_userdata/rubygem-foreman_userdata.spec]: This should be https

File: rubygem-foreman_userdata/rubygem-foreman_userdata.spec [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+Source0: http://rubygems.org/gems/%{gem_name}-%{version}.gem
```
Comment (ekohl) [inline rubygem-foreman_userdata/rubygem-foreman_userdata.spec]: This too

**Comment by GregSutcliffe** [comment]
Was this only merged to develop? The request is for 1.14+

File: rubygem-foreman_userdata/rubygem-foreman_userdata.spec (hunk 1 above)
Comment (ekohl) [reply rubygem-foreman_userdata/rubygem-foreman_userdata.spec]: https://github.com/theforeman/foreman-packaging/pull/1811

**Pull request merged**